src/pages/Markets/Recommended.tsx

```diff
@@ -175,6 +178,52 @@ const OneClickDefiAssets: React.FC<{
   )
 }
 
+const ThorchainAssets: React.FC<{
+  selectedChainId: ChainId | undefined
+}> = ({ selectedChainId }) => {
+  const dispatch = useAppDispatch()
+  const { data: thorchainAssetIdsData, isLoading: isThorchainAssetIdsDataLoading } = useQuery({
+    queryKey: ['thorchainAssets'],
+    queryFn: thorchainSaversOpportunityIdsResolver,
+    staleTime: Infinity,
+    select: pools => pools.data,
+  })
+
+  useEffect(() => {
+    ;(async () => {
+      await dispatch(
+        opportunitiesApi.endpoints.getOpportunityIds.initiate(
+          {
+            defiType: DefiType.Staking,
+            defiProvider: DefiProvider.ThorchainSavers,
+          },
+          { forceRefetch: true },
+        ),
+      )
+
+      await dispatch(
+        opportunitiesApi.endpoints.getOpportunitiesMetadata.initiate(
+          [
+            {
+              defiType: DefiType.Staking,
+              defiProvider: DefiProvider.ThorchainSavers,
+            },
+          ],
+          { forceRefetch: true },
+        ),
+      )
+    })()
+  }, [dispatch])
```

Comment:
Fetches savers data since we normally lazy fetch on earn tab

Reply:
![image](https://github.com/user-attachments/assets/d1f0da6e-47ce-45ce-8612-433fcca6050a)

Any reasons why we don't have the APYs there?

Reply:
Seems happy to me - can take a sec or two to load but by the time you scroll should be g2g. As discussed, not added because RTK loading states are a 🥙 
<img width="1397" alt="image" src="https://github.com/user-attachments/assets/8dd984fb-0fc1-4643-8e22-675b74012b63">